packages/neuron-ui/src/components/Overview/index.tsx

```diff
@@ -34,16 +34,26 @@ import { backToTop } from 'utils/animations'
 const TITLE_FONT_SIZE = 'xxLarge'
 export type ActivityItem = State.Transaction & { confirmations: string; typeLabel: string }
 
-const genTypeLabel = (type: 'send' | 'receive', confirmationCount: number) => {
+const genTypeLabel = (
+  type: 'send' | 'receive',
+  confirmationCount: number,
+  status: 'pending' | 'success' | 'failed'
+) => {
   switch (type) {
     case 'send': {
-      if (confirmationCount < CONFIRMATION_THRESHOLD) {
+      if (status === 'failed') {
```

Comment:
This seems to be the same as the fallback (last line of return).

Reply:
All tx are remarked as pending without this branch when the tip block number is less than or equal to the confirmations threshold.